dub.json

```diff
@@ -46,25 +46,35 @@
             "name": "test",
             "mainSourceFile": "test.d",
             "targetName": "test",
-            "sourceFiles": [ "tests/unit/HttpClient.d" ],
+            "sourceFiles": [ "tests/HttpClient.d" ],
             "excludedSourceFiles": ["dstep/*", "clang/*"],
         },
 
         {
             "name": "unittest",
-            "mainSourceFile": "unittest.d",
             "targetName": "unittest",
+            "mainSourceFile": "bin/ut.d",
+            "sourceFiles": ["tests/Common.d", "tests/Assert.d"],
             "sourcePaths": ["dstep", "clang", "tests/unit"],
             "excludedSourceFiles": ["dstep/main.d"],
 
             "preBuildCommands-posix": [
-                "(! [ -s linker_flags.txt ] && ./configure) || true"
+                "(! [ -s linker_flags.txt ] && ./configure) || true",
+                "dub run unit-threaded -c gen_ut_main -- -f bin/ut.d"
+            ],
+
+            "preBuildCommands-windows": [
```

Comment:
Can the platform suffix be removed to avoid duplicating line [63](https://github.com/jacob-carlborg/dstep/pull/182/files#diff-c440c818ecdf6795a339ff0ad2fba6a0R63) and [67](https://github.com/jacob-carlborg/dstep/pull/182/files#diff-c440c818ecdf6795a339ff0ad2fba6a0R67)?

Reply:
Will we have a guarantee that they will be executed in desired order?

Reply:
Do you mean the order of the two commands on Posix? I don't think it matters which order those are executed in.

Reply:
Fixed.